Druid-Tests/DRProductionPrimitiveCompilationTest.class.st

```diff
@@ -24,24 +24,30 @@ DRProductionPrimitiveCompilationTest >> setUp [
 DRProductionPrimitiveCompilationTest >> testCompilePrimitiveAt [
 
 	| class array |
+	(cogit isKindOf: SimpleDruidTestRTLCompiler) ifTrue: [ ^ self skip ].
+
```

Comment:
I skipped several tests like this. The reason is that `primitiveAt` has spill slots, but the access to the arguments in the stack is not aware of it. The code reads like

```
"Allocate 3 spill slots in the stack"
self SubCq: 24 R: SPReg.
self genLoadArg0R: R1

=>
genLoadArg0R: destinationRegister
    self MoveMw: 'Fixed offset that does not take the spill slots into account' r: SPReg R: destinationRegister.
```